doc/installation/index.rst

```diff
@@ -273,6 +301,13 @@ following commands ::
   $ cd /path_to_centreon_broker/build
   $ cmake .
 
+.. note::
+    If you are using Debian Strech or Ubuntu Xenial, you need to add the following
+    statement in the **CMakeLists.txt** file (directory `build`), use the command
+    bellow to add ::
+
+      $ sed -i '26iset(CMAKE_CXX_FLAGS "-std=c++98 -fpermissive")' CMakeLists.txt
```

Comment:
I dislike this solution that overwrite CMake compilation flags. I would rather use the CXXFLAGS environment variable to add specific flags. This has to be run before the cmake command below. -fpermissive should not be needed but -Wno-long-long might as this should be the sole exception to the standard.

`export CXXFLAGS="-std=c++98 -Wno-long-long"`

Reply:
Yes, i agree. really is a lot more elegant than running a sed in Cmake file.